source/blog/2010-08-07-dc2-experimental-associations-id-fields.md

```diff
@@ -33,10 +33,10 @@ sure that it works correctly and does not have to many problematic
 edge-cases. Therefore we need your feedback.
 
 -   Go to
-    [[https://github.com/doctrine/doctrine2/commits/DDC-117](https://github.com/doctrine/doctrine2/commits/DDC-117)](https://github.com/doctrine/doctrine2/commits/DDC-117)
+    [[https://github.com/doctrine/orm/commits/DDC-117](https://github.com/doctrine/orm/commits/DDC-117)](https://github.com/doctrine/orm/commits/DDC-117)
     to see the code
 -   [Have a look at the functional
-    tests](https://github.com/doctrine/doctrine2/blob/DDC-117/tests/Doctrine/Tests/ORM/Functional/Ticket/DDC117Test.php)
+    tests](https://github.com/doctrine/orm/blob/DDC-117/tests/Doctrine/Tests/ORM/Functional/Ticket/DDC117Test.php)
```

Comment:
404. Maybe we should use the link of the master branch here: `https://github.com/doctrine/orm/blob/master/tests/Doctrine/Tests/ORM/Functional/Ticket/DDC117Test.php`